packages/adminrouter/docker/build-resty.sh

```diff
@@ -9,6 +9,7 @@ cd $OPENRESTY_DIR
     "--prefix=$AR_BIN_DIR" \
     --with-file-aio \
     --with-http_gunzip_module \
+    --with-http_gzip_module \
```

Comment:
> 10:56:14 ./configure: error: invalid option "--with-http_gzip_module"

For the record: I don't think we need that, enabled by default. Wasn't needed in https://github.com/dcos/dcos/pull/3281 to get gzip compression working.

Reply:
do we even need `http_gunzip_module` then?